### PR TITLE
show neighborhood instances

### LINKED ISSUE #1844: create a component for place based instances
## acceptance criteria

- [ ] requires individual "card" components
- [ ] larger "find your neighborhood" component should accept any number of cards as children
- [ ] used on `/find-placecal` with all neighborhoods
- [ ] used on `/` with 4 selected neighborhoods

## implementation plan

The bottom part of this mock up "check out a neighborhood"

Don't include the lorem filled text as discussed in meeting

image

see #1818 for list of homepage neighborhoods

### PR DESCRIPTION
**Pull request by aaaaargZombies**
fixes #1844

## description

- created 3 components
    - container with coloured header
    - a grid that's 4 columns at desktop
    - a card for previewing neighbourhoods
- pull published sites into the pages controller (we will need to update production to get the correct sites showing)
- add elements to displays sites on "home" and "find your placecal" pages

Screen shots

Screenshot 2023-05-11 at 12-11-38 PlaceCal The Community Calendar

Screenshot 2023-05-11 at 12-12-04 PlaceCal The Community Calendar

Screenshot 2023-05-11 at 12-12-43 PlaceCal The Community Calendar

## questions

- I'm expecting feedback from Mark for design tweaks.
- I'm not convinced that "visit" is the best thing to have in the link text but I don't think it's broken.

### CODE REVIEW

**Comment by studiosquid** [comment]
If time permits could we look at the padding at the left hand and right hand sides on the desktop state. My bad, I haven't provided clear guidance on this — the underlying grid is hidden in the XD previews (sigh).
I've screenshotted my XD file for this _(you can see I abandoned following my own grid structure for the actual four neighbourhood blocks but the outer edge padding is a bit of a rule of thumb on most desktop components)_. If time doesn't permit it can be a future refinement.

Also a touch more vertical spacing around the div / line the button feels very close to it. (This is nitpick territory, only if quick and have time.)

Agree the button text isn't quite right — for now maybe it just needs to be longer "Visit site" or "View site" also could the button be a bit longer even without the text length within it? Feels too small right now.

The rest looks good I think — though I'm not sure the "Check out a neighbourhood" title will work at mobile state, I'd expect that title to run to two lines — but I'm not sure that last screenshot is the mobile state as the logo looks quite small? 

**Screenshot of Desktop size:** 
Screenshot 2023-05-11 at 16 50 15

**Comment by studiosquid** [comment]
I've also realised "Check out a neighbourhood" is my copy so this might be one for Honor to take a look at but that might be a bit zoomed in and we leave it as this for now.

File: app/components/home_strapline_component/home_strapline_component.sass [hunk 1]
```diff
@@ -5,7 +5,6 @@
   border-radius: $default-radius
   color: $base-text
   font-family: $serif
-  margin: 1.5rem
```
Comment (r-ferrier) [inline app/components/home_strapline_component/home_strapline_component.sass:8]: lol I have done literally the exact same thing in my PR

**Review by r-ferrier — APPROVED** [review]
From a code perspective looks good - and a test!

**Comment by aaaaargZombies** [comment]
I'll open a ticket for the title and we can deal with it then

**Pull request merged**